tuf/tuf.go

```diff
@@ -173,6 +173,11 @@ func (tr *Repo) RemoveBaseKeys(role string, keyIDs ...string) error {
 	return nil
 }
 
+// GetAllLoadedRoles returns a list of all role entries loaded in this TUF repo, could be empty
+func (tr *Repo) GetAllLoadedRoles() []*data.Role {
```

Comment:
Looking at the current code, this just makes me realize even more we need to merge /client/client.go and /tuf/client/client.go. The one in /tuf keeps a reference to the keyDB, the other doesn't :-(